src/dbt/kipptaf/models/amplify/mclass/intermediate/int_amplify__benchmark_student_summary_unpivot.sql

```diff
@@ -1,96 +1,126 @@
-                decoding_nwf_wrc_score,
-                decoding_nwf_wrc_semester_growth,
-                decoding_nwf_wrc_year_growth
-            ) as 'Decoding (NWF-WRC)',
-            (
-                letter_names_lnf_level,
-                letter_names_lnf_national_norm_percentile,
-                letter_names_lnf_score,
-                letter_names_lnf_semester_growth,
-                letter_names_lnf_year_growth
-            ) as 'Letter Names (LNF)',
-            (
-                letter_sounds_nwf_cls_level,
-                letter_sounds_nwf_cls_national_norm_percentile,
-                letter_sounds_nwf_cls_score,
-                letter_sounds_nwf_cls_semester_growth,
-                letter_sounds_nwf_cls_year_growth
-            ) as 'Letter Sounds (NWF-CLS)',
-            (
-                phonemic_awareness_psf_level,
-                phonemic_awareness_psf_national_norm_percentile,
-                phonemic_awareness_psf_score,
-                phonemic_awareness_psf_semester_growth,
-                phonemic_awareness_psf_year_growth
-            ) as 'Phonemic Awareness (PSF)',
-            (
-                reading_accuracy_orf_accu_level,
-                reading_accuracy_orf_accu_national_norm_percentile,
-                reading_accuracy_orf_accu_score,
-                reading_accuracy_orf_accu_semester_growth,
-                reading_accuracy_orf_accu_year_growth
-            ) as 'Reading Accuracy (ORF-Accu)',
-            (
-                reading_comprehension_maze_level,
-                reading_comprehension_maze_national_norm_percentile,
-                reading_comprehension_maze_score,
-                reading_comprehension_maze_semester_growth,
-                reading_comprehension_maze_year_growth
-            ) as 'Reading Comprehension (Maze)',
-            (
-                reading_fluency_orf_level,
-                reading_fluency_orf_national_norm_percentile,
-                reading_fluency_orf_score,
-                reading_fluency_orf_semester_growth,
-                reading_fluency_orf_year_growth
-            ) as 'Reading Fluency (ORF)',
-            (
-                word_reading_wrf_level,
-                word_reading_wrf_national_norm_percentile,
-                word_reading_wrf_score,
-                word_reading_wrf_semester_growth,
-                word_reading_wrf_year_growth
-            ) as 'Word Reading (WRF)',
-            (
-                composite_level,
-                composite_national_norm_percentile,
-                composite_score,
-                composite_semester_growth,
-                composite_year_growth
-            ) as 'Composite'
-        )
+            case
+                measure
+                when 'Composite'
+                then 'Composite'
+                when 'Reading Comprehension (Maze)'
+                then 'Comprehension'
+                else substr(measure, strpos(measure, '(') + 1, 3)
```

Comment:
`regexp_extract` can do this more reliably

https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#regexp_extract